src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './styles/App.css';
+
+class App extends Component {
+	render() {
+		return (
+			<div className="App">
+				<h5>Welcome to Authors Heaven</h5>
```

Comment:
Typo error on Authors Heaven which should be changed to Author's Haven

Reply:
Thanks